app/assets/javascripts/application.js

```diff
@@ -15,13 +15,38 @@
 //= require bootstrap-sprockets
 //= require_tree .
 
+var checkBox = "input[type=checkbox]";
+var checked = "input[type=checkbox]:checked";
+
+// filters the suppliers items from checkbox clicked
+var $filterSupplierItems =     $(".supplier_list" + '' + checkBox).click(function(){
+  $('.suppliers').hide();
+  var $selectedSupplier = $(checked).map(function(index, element){
+    return '.' + $(element).val()
+  });
+  var suppliers = $.makeArray($selectedSupplier).join(",");
+  suppliers ? $(suppliers).show() : $('.suppliers').show();
```

Comment:
Mixed double and single quotes.<br>Expected an assignment or function call and instead saw an expression.